exporter/datadogexporter/internal/metrics/consumer.go

```diff
@@ -35,18 +36,22 @@ var _ translator.APMStatsConsumer = (*Consumer)(nil)
 // Consumer implements translator.Consumer. It records consumed metrics, sketches and
 // APM stats payloads. It provides them to the caller using the All method.
 type Consumer struct {
-	ms        []datadogV2.MetricSeries
-	sl        sketches.SketchSeriesList
-	as        []pb.ClientStatsPayload
-	seenHosts map[string]struct{}
-	seenTags  map[string]struct{}
+	ms                          []datadogV2.MetricSeries
+	sl                          sketches.SketchSeriesList
+	as                          []pb.ClientStatsPayload
+	seenHosts                   map[string]struct{}
+	seenTags                    map[string]struct{}
+	sumAsRate                   bool
+	sumToRateConversionInterval time.Duration
 }
 
 // NewConsumer creates a new Datadog consumer. It implements translator.Consumer.
-func NewConsumer() *Consumer {
+func NewConsumer(metricsSumAsRate bool, metricsSumToRateConversionInterval time.Duration) *Consumer {
```

Comment:
If this implements translator.Consumer, does that mean the method signature upstream needs to be updated as well to match this?